ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywallView.kt

```diff
@@ -40,9 +43,24 @@ internal fun InternalPaywallView(
                 Text(text = "Error: ${state.errorMessage}")
             }
             is PaywallViewState.Loaded -> {
-                Text(text = "Paywall for offeringId: ${state.offering.identifier}")
+                val offering = state.offering
+                Text(text = "Paywall for offeringId: ${offering.identifier}")
+                // TODO-PAYWALLS: Use device locale and move logic to view model
+                offering.paywall?.configForLocale(Locale.ENGLISH)?.let { localizedConfiguration ->
+                    Text(
+                        style = MaterialTheme.typography.h5,
+                        textAlign = TextAlign.Center,
+                        text = localizedConfiguration.title,
+                    )
+                    Text(
+                        style = MaterialTheme.typography.subtitle1,
+                        textAlign = TextAlign.Center,
+                        text = localizedConfiguration.subtitle ?: "",
+                    )
```

Comment:
Should we not build the subtitle View at all if there's no subtitle?

Reply:
The specifics might be different in Compose, but one reason why a lot of the time I do display the `Text`s even if they're empty is to make layouts consistent.

When we implement [intro eligibility](https://linear.app/revenuecat/issue/PWL-204/reusable-view-to-display-different-text-depending-on-intro-eligibility), the content of these labels might change depending on the selected package, and sometimes that would be empty (for example you can have an empty `offer_details` but not-empty `offer_details_with_intro_offer`).